src/pages/index.js

```diff
@@ -1,21 +1,29 @@
 import logo from "../images/logo.svg";
+
 import avatar from "../images/avatar.jpg";
 import pencil from "../images/pencil.svg";
 import plus from "../images/plus.svg";
+import pencilEditAvatar from "../images/pencil-edit-avatar.svg";
+import closeIcon from "../images/close-icon-light.svg";
 
 document.addEventListener("DOMContentLoaded", () => {
   document.querySelector(".header__logo").src = logo;
   document.querySelector(".profile__avatar").src = avatar;
```

Comment:
`document.querySelector(".profile__avatar")` 

It’s better to find **constant DOM elements** only 1 time at the top of the file so as not to waste resources on searching them again and again when you call a method (function) because searching is a very **hard** operation for the browser engine

Reply:
My remark hasn't been corrected